Crea la tarea de la clase 5

Lenz-or opened the pull request:
Se realizo la tarea de la clase

Lenz-or committed:
Crea la tarea de la clase 5

Se realizo la tarea de la clase

fsodano commented on tareas/clase-5/tarea-clase-5.js:
```diff
@@ -29,6 +29,23 @@ $botonCalcular.onclick = function()
+}
+
+function calcularSalarioMensual(salarioAnual){
+  const MESES_AÑO = 12;
```
no ñ ni caracteres no estándares en los nombres de variables y funciones

Lenz-or committed:
Cambia y arregla la tarea 5

Agrega espacios al formulario y luego se hace un cambio en el codigo de JS siguiendo las refrencias de la tarea

Lenz-or committed:
Arregla codigo

Cambio la ñ por n
